### ISSUE TITLE
No such module 'BigInt'

### ISSUE THREAD

**Issue by BitLox** [AlphaWallet/alpha-wallet-ios#2135]
Cloned and attempted to compile.

Pops an error 
`No such module 'BigInt'`
for the Swift compiler in Xcode

Xcode Version 11.7 (11E801a)
macOS 10.15.6

**Comment by hboon** [comment]
>Cloned and attempted to compile. ...

@BitLox thanks. Will check. 

For the time being, try Xcode 11.3. We use that internally.

**Comment by hboon** [comment]
Related to #1831

**Comment by pekkanikander** [comment]
I'm seeing this, too, now, with Monterey 12.1 and XCode 13.2.1.  I've never used CocoaPods before, so I have hard time figuring out why Swift does not seem to find the pods.  The pods themselves seem to get properly installed with `make bootstrap`

**Comment by eviltofu** [comment]
@BitLox @pekkanikander are you opening the Xcode workspace file or the project file?

**Comment by pekkanikander** [comment]
Thanks @eviltofu !  That was it.  Opening the workspace file instead of the project worked nicely.  

The reason I had switched to the project file was that initially I had a lot of problems in getting the BUNDLE_INDENTIFIERs and App Groups working.  Being not that experienced with modern Apple Concepts, I thought fixing them in the project would be easier.  (I started developing for the Mac already in the OS9 days, but I have had now a break of a couple of years in developing for the Mac and iOS.)

I was able to fix the identifier and group issues, but this one remained.  I failed to remember that I had changed to working on the project file and not with the workspace.  Thanks for your help!
